rancher2/resource_rancher2_cloud_credential.go

```diff
@@ -136,6 +136,8 @@ func resourceRancher2CloudCredentialUpdate(d *schema.ResourceData, meta interfac
 		update["s3credentialConfig"] = expandCloudCredentialS3(d.Get("s3_credential_config").([]interface{}))
 	case vmwarevsphereConfigDriver:
 		update["vmwarevspherecredentialConfig"] = expandCloudCredentialVsphere(d.Get("vsphere_credential_config").([]interface{}))
+	case openTelekomCloudConfigDriver:
+		update["opentelekomcloudcredentialConfig"] = expandCloudCredentialOpenTelekomCloud(d.Get("open_telekom_cloud_credential_config").([]interface{}))
```

Comment:
please use one word: opentelekomcloud_credential_config

Reply:
Sure, changed it.